big-practice/src/js/pages/login.page.js

```diff
@@ -0,0 +1,100 @@
+import { formValidate } from '../validates/form.validate';
+import { apiService } from '../services/api.service';
+import { DocumentHelper } from '../helpers/show-message.helper';
+import { ACCOUNTS_API } from '../constants/url-api.constant';
+import { MESSAGES, EMPTY_TEXT } from '../constants/message.constant';
+import { DASHBOARD_PAGE, USERNAME_KEY } from '../constants/app.constant';
+
+class Login {
+  constructor() {
+    // Query elements
+    this.emailInput = document.getElementById('email');
+    this.passwordInput = document.getElementById('password');
+    this.loginBtn = document.getElementById('loginBtn');
+    this.generalWarnMsg = document.querySelector('.general-warn-msg');
+  }
+
+  /**
+   * Handling account login to dashboard
+   */
+  async login() {
+    const data = {
+      email: this.emailInput.value,
+      password: this.passwordInput.value,
+    };
+    const config = {
+      email: ['empty', 'formatEmail'],
+      password: ['empty'],
+    };
+    const validate = formValidate.validateForm(data, config);
+    const url = `${ACCOUNTS_API}?email=${data.email}&password=${data.password}`;
+
+    if (!validate.isValid) {
+      DocumentHelper.showErrorMessage(this.emailInput, validate.errors.email);
+      DocumentHelper.showErrorMessage(this.passwordInput, validate.errors.password);
+
+      return;
+
+    }
+
+    const movieList = await apiService.get(url);
```

Comment:
movieList???
What is the response of this api call?

Reply:
updated!